### PR TITLE
Add web authentication pages (Login and Register)

### PR DESCRIPTION
**Pull request by arnoldkunihira**

### CODE REVIEW

File: src/components/Login.vue [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                            class="form-control" 
+                            id="floatingPassword" 
+                            placeholder="Password"
+                            v-model="password">
+                        <label for="floatingPassword">Password</label>
+                    </div>
+                
+                    <div class="checkbox mb-3">
+                        <label>
+                            <input type="checkbox" value="remember-me"> Remember me
```
Comment (vongohren) [inline src/components/Login.vue:50]: Do you need this as you are not handling cookies yourself?

File: src/components/Register.vue [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                name: this.name, 
+                email: this.email, 
+                password: this.password,
+                password_confirmation: this.password_confirmation
+            }).then(function (response) {
+                const data = response.data;
+                console.log(data);
+                
+                if (data.status === 'success') {
+                    currentObject.showSuccess = true;
```
Comment (vongohren) [inline src/components/Register.vue:104]: Not sending to dashboard when success? No need to build out the email verfication. 😅

File: src/utils/requests.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+const PIZZERIA_HOST = '';
```
Comment (vongohren) [inline src/utils/requests.js:3]: How do you override this?

File: src/components/Login.vue (hunk 1 above)
Comment (arnoldkunihira) [reply src/components/Login.vue:50]: I actually do not need it, as handling cookies right now is not one of the functional items.

File: src/components/Register.vue (hunk 2 above)
Comment (arnoldkunihira) [reply src/components/Register.vue:104]: This I agree, however, the logic when a client clicks the **proceed to checkout** button should be, they are redirected to log in if they have an account or register if they don't have an account, and once they are able to log in, they can view the **order summary** that is scrollable depending on the number of products within the order and also add **delivery details** then the order summary will have (cost of items and a delivery cost) and also a section of pay using (Mobile money, Visa or any other payment method which prompts them to add their mobile number, and confirm their order (Confirm your order button). That is how my thought process would be to actually complete the order until the client gets their product/pizza.

File: src/utils/requests.js (hunk 3 above)
Comment (arnoldkunihira) [reply src/utils/requests.js:3]: Ideally, I can override it by changing the environment am using from development to production, which means the ternary conditional operator is modified to suit my needs.

File: src/components/Register.vue (hunk 2 above)
Comment (vongohren) [reply src/components/Register.vue:104]: Good suggeestion

**Pull request merged**